### ISSUE TITLE
The Tabs Widget cannot be activated using NVDA screen reader while navigating in brows mode

### ISSUE THREAD

**Issue by bpmohith** [jquery/jquery-ui#2365]
The Tabs Widget cannot be activated using NVDA screen reader while navigating in brows mode

**Steps to Reproduce:**
1. Open [Tabs Widget](https://api.jqueryui.com/tabs/#entry-examples ) Example on Chrome Browser.
2. Turn on NVDA screen reader.
3. Start navigating with the 'h' quick navigation key till the NVDA focus reaches the Example > Demo heading.
4. Start navigating with down arrow. When 'One' tab is active press down arrow till the NVDA focus moves to the 'Two' tab.
5. Press enter and space key and observe.

**Description:**
NVDA is a screen reader. Users with blindness use to operate computer. All UI elements are required to be operable with screen reader. In the current scenario a blind screen reader user cannot activate tabs when navigating with the NVDA brows mode.
The tabs are actionable only with the focus mode. This causes lots of issues to screen reader users and many will not know how to switch from brows to focus mode just to activate a tab.
The [APG Tab Example](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/examples/tabs-automatic/) works fine with NVDA brows mode.
NVDA screen reader can be downloaded from:
https://www.nvaccess.org/download/

**Comment by mgol** [comment]
Thanks for the report. I was able to confirm this on a Windows 11 VM with NVDA. VoiceOver on macOS doesn't have the issue.

This is something that I'd definitely want to fix.
